src/lib/containers/demo_components/Login.tsx

```diff
@@ -84,6 +84,7 @@ class Login extends React.Component<Props, State> {
     clickEvent.preventDefault() // avoid page refresh
     SynapseClient.login(this.state.username, this.state.password)
             .then((data: any) => {
+              SynapseClient.setSessionTokenCookie(data.sessionToken)
```

Comment:
Would this need a check at all? Will setSessionTokenCookie ever fail? Or is the proof of concept?

Reply:
should not need to (response is empty, but instructs browser to set/clear cookie)